.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: iodide/pyodide-env:11
+    - image: iodide/pyodide-env:12
```

Comment:
You need to first push to some temporary path to test that it's working. And then once the CI passes push that image to this location and update the path.

Could you send me your Docker Hub username, so I can give you write permission to that Docker org?

Reply:
I have "hoodmane" at Docker Hub too.

> You need to first push to some temporary path to test that it's working. And then once the CI passes push that image to this location and update the path.

I don't understand this.

Reply:
> I have "hoodmane" at Docker Hub too.

Hmm, looks like we can have 3 people max in https://hub.docker.com/orgs/iodide presumably under the free plan. We probably need to request OSS support for https://hub.docker.com/orgs/pyodide but   that can take some time.

> I don't understand this.

You can,
1. Build the image locally as `hoodmane/pyodide-env:12` (or some other version) and push it there.
2. Update the path in this PR to `hoodmane/pyodide-env:12`, make sure the CI works
3. Now push that image to `iodide/pyodide-env:12` (in this case I'll probably have to do it, while we figure out Docker hub org size issue)
4. Update the path in the PR back to `iodide/pyodide-env:12`

It's a bit tedious but I haven't found a better solution so far.

Reply:
Okay I did steps 1+2.